### ISSUE TITLE
Could not find executable pandoc-mermaid on Linux

### ISSUE THREAD

**Issue by YingYoh09** [raghur/mermaid-filter#49]
I can't run pandoc on ubuntu 18.04

pandoc --version: `2.9.2.1`

Installed with: `sudo npm install -g mermaid-filter --unsafe-perm=true`
Because I can't install with `sudo npm install -g mermaid-filter` or `npm install -g mermaid-filter`

When I run: `pandoc  -F pandoc-mermaid -o readme.pdf readme.md`
> Error running filter pandoc-mermaid:
Could not find executable pandoc-mermaid

Thx

**Comment by japorized** [comment]
The filter name is actually `mermaid-filter` instead of `pandoc-mermaid`. Try this instead.
```
pandoc -F mermaid-filter -o readme.pdf readme.md`
```

**Comment by YingYoh09** [comment]
Thanks

**Comment by qkzk** [comment]
Hello,

I'm still having the same issue on arch :

```sh
% npm install --global mermaid-filter

changed 170 packages, and audited 171 packages in 12s

15 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
% pandoc -F mermaid-filder -o test.pdf
Error running filter mermaid-filder:
Could not find executable mermaid-fildertest.md
% sudo npm install -g mermaid-filter --unsafe-perm=true
[sudo] password for :

changed 170 packages, and audited 171 packages in 15s

15 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
npm notice
npm notice New patch version of npm available! 7.11.1 -> 7.11.2
npm notice Changelog: https://github.com/npm/cli/releases/tag/v7.11.2
npm notice Run npm install -g npm@7.11.2 to update!
npm notice
% pandoc -F mermaid-filter -o test.pdf test.md
```

Is there something I'm missing ?

Thanks in advance

**Comment by qkzk** [comment]
Solved it with :

```
% npm install mermaid-filter
% pandoc -F ~/node_modules/mermaid-filter/index.js test.md -o test.pdf
```
